src/Json/JsonGenerator.php

```diff
@@ -179,6 +179,10 @@ protected function createDebit($transaction, $language){
             $data['l2l3Data'] = $this->stringifyL2L3Data($l2l3Data);
         }
 
+        if ($transaction->getIncludeTracing()) {
+            $data['includeTracing'] = $transaction->getIncludeTracing();
+        }
+
```

Comment:
Also for this, you can add a method that handles setting the tracing, so changes won't have to be applied twice.